django/cantusdb_project/main_app/views/views.py

```diff
@@ -620,3 +625,40 @@ def content_overview(request):
     return render(
         request, "content_overview.html", {"objects": recently_updated_50_objects}
     )
+
+def redirect_node_url(request, pk):
```

Comment:
Can we have a docstring here, briefly outlining why we need this view: backwards compatibility, different numbering systems, and so on?